changelog/seqera-enterprise/v24.1_major.mdx

```diff
@@ -0,0 +1,96 @@
+---
+title: Seqera Enterprise v24.1_major
+date: 2024-07-04
+tags: [seqera enterprise]
+---
+
+Seqera Platform Enterprise version 24.1 introduces three new features: Data Studios (in public preview), Data Explorer, and managed identities. A number of bug fixes and performance enhancements have also been included in this major release.
+
+## Data Studios
+
+[Data Studios](https://docs.seqera.io/platform/24.2/data_studios) closes the loop from development to deployment and insights, allowing you to create, manage, and share notebook environments in Seqera with the click of a button. Data Studios also makes it seamless to work across teams with multi-user support, built-in authentication, and automatic snapshots as you work.
+
+- Host a combination of container images and compute environments for interactive analysis using your preferred tools, like Jupyter notebooks, RStudio, and Visual Studio Code IDEs.
+- Checkpoints provide a stable point-in-time snapshot.
+- Collaborate and share data in real time.
+
+Data Studios is available as a public preview. [Contact support](https://support.seqera.io) to request access to this feature. 
```

Comment:
We're using this in docs

```
:::note
Data Studios is currently in **private preview** and is not yet generally available. To request access, please contact your Seqera account executive.
:::
```

Reply:
Let's update the docs to reflect that it's in public preview.

Reply:
True. But also different wording:

**To request access, please contact your Seqera account executive.**